### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by Juank628**
## To-Do List: initial setup
In this pull request I did the following:
- Setup linters
- Setup Webpack
- Created HTML template
- Created to-do list render logic

### CODE REVIEW

**Review by andersonlebon — CHANGES_REQUESTED** [review]
### Hello @Juank628 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
# Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

File: README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```
Comment (andersonlebon) [inline README.md:10]: - [ ] Kindly make sure you removed the template logo. Optionally, you can replace it with your own logo if you have one.
Screenshot from 2022-12-07 03-36-21

File: README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+This page registers books with author and title of the book. To register a book:
+1. Type the author's name.
+2. Type the book's title.
+3. Click on Add button below.
+4. The books will be added to the list and will be displayed.
+
+To remove a book from the list:
+1. Find the book in the list.
+2. Click on remove button for the respective book.
```
Comment (andersonlebon) [inline README.md:100]: - [ ] By using these steps, we can't able to run this project. Kindly make sure you gave all information about how to run this To-Do list app. :smiley:

File: README.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+Clone this repository to your desired folder:
+
+git clone git@github.com:Juank628/to_do_list.git
+
+### Install
+
+Install this project with:
+
+npm install
+
```
Comment (rloterh) [inline README.md:87]: - [ ] Kindly also include instructions here for your repo visitors, under the _Getting started_, on how to run your app. For instance, doing so with the command: `npm start`, `npx webpack` etc, as informed in the previous review.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+3. The activity will be added to the list and will be displayed.
+
+To remove a book from the list:
+1. Find the activity in the list.
+2. Click on the more button for the respective activity.
+3. Click on the delete button.
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
```
Comment (rloterh) [inline README.md:102]: - [ ] OPTIONAL
I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Juank628/to_do_list/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
```
Comment (rloterh) [inline README.md:132]: - [ ] OPTIONAL
It's recommended as good practice that you add an MIT license, or any other similar license to your ReadMe for this project.

File: README.md [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
```
Comment (rloterh) [inline README.md:32]: - [ ] OPTIONAL
Some aspect(s) of your table of content do(es) not navigation to the appropriate content on your ReadMe, kindly endeavor to fix it.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Juanca, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Project is set up correctly

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @Juank628 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job working on the changes provided by the previous reviewers ✅
- Professional README.md file 💯


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**